blocks/api/categories.js

```diff
@@ -11,7 +14,7 @@ import { __ } from '@wordpress/i18n';
  *
  * @var {Array} categories
  */
-const categories = [
+let categories = [
```

Comment:
note that there's no need to change this to a `let` just to allow for `push()` and `splice()` and `sort()` - those are all mutating operations. not a big deal, but just pointing out that we don't need to introduce this change